### ISSUE TITLE
Using morphodita with nametag raises exception

### ISSUE THREAD

**Issue by Mimino666** [ufal/morphodita#7]
I tried to to use both *morphodita* and *nametag* at the same time, but a strange exception is raised, when I import *nametag* into the project:

```python
from ufal.morphodita import Tagger, Forms, TokenRanges
import ufal.nametag  # this line causes the problem

forms = Forms()
tokens = TokenRanges()
tagger = Tagger.load(path_to_model)
tokenizer = tagger.newTokenizer()
tokenizer.setText(text)
while tokenizer.nextSentence(forms, tokens):
    pass
```

Result:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: in method 'Tokenizer_nextSentence', argument 2 of type 'std::vector< std::string,std::allocator< std::string > > *'
```

If I switch the order of the imports, the exception is not raised in the above example, but if I try to use *nametag* a different exception is raised.

**Comment by foxik** [comment]
Confirmed.

Will look into it sometimes during the week.

**Comment by foxik** [comment]
The problem was caused by the way how NameTag was using MorphoDiTa.

I released new NameTag version (1.1.1) and new CPAN and PyPI packages.

I am closing the issue now, if it still persists (which it should not), please reopen it.

**Comment by Mimino666** [comment]
Looks okay. Thank you for a quick fix :)
